sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroEncoder.cs

```diff
@@ -32,6 +32,7 @@ public class SchemaRegistryAvroEncoder
         private readonly string _groupName;
         private readonly SchemaRegistryAvroObjectEncoderOptions _options;
         private const string AvroMimeType = "avro/binary";
+        private const string ContentTypeKey = "ContentType";
```

Comment:
nit:  If you're defining this on the message with metadata dictionary thing,  we should consider using that rather than redefining.